src/test/java/org/eclipse/cargotracker/application/BookingServiceTest.java

```diff
@@ -84,17 +82,25 @@
 // dates.
 @RunWith(Arquillian.class)
 public class BookingServiceTest {
-
-    @Inject private BookingService bookingService;
-    @PersistenceContext private EntityManager entityManager;
-
+    private static final Logger LOGGER = Logger.getLogger(BookingServiceTest.class.getName());
     private static TrackingId trackingId;
     private static List<Itinerary> candidates;
-    private static Date deadline;
+    private static LocalDate deadline;
     private static Itinerary assigned;
+    @Inject private BookingService bookingService;
+    @PersistenceContext private EntityManager entityManager;
 
     @Deployment
     public static WebArchive createDeployment() {
+        File[] extraJars =
+                Maven.resolver()
+                        .loadPomFromFile("pom.xml")
+                        .resolve("org.apache.commons:commons-lang3", "org.postgresql:postgresql")
```

Comment:
As discussed, kindly remove PostGRESQL for now.

Reply:
updated.